doc/source/reference/credits.rst

```diff
@@ -263,7 +263,8 @@ Matthew Palathingal
 Kelly Wang
 
 * Enabled NeighborList indexing.
-* Added methods ``compute_distances`` and ``compute_all_distances`` to Box.
+* Added methods ``compute_distances``, ``compute_all_distances`` to Box.
```

Comment:
If I'm not mistaken, using "and" instead of a comma is correct grammar. Why was this change made?

Reply:
Oh that was a typo. My bad.